.github/workflows/openssl_versions.yml

```diff
@@ -0,0 +1,153 @@
+---
+name: OpenSSL Versions
+
+on:
+  pull_request:
+    branches: ["main"]
+  push:
+    branches: ["main"]
+
+jobs:
+  build:
+    name: Versions
+    runs-on: ubuntu-22.04
```

Comment:
Given that we run everything in containers, should we change to `ubuntu-latest`? The 22.04 will eventually go EOL. This is issue also in the other workflow files. Not a big deal though.

Reply:
last time I tried ubuntu:latest things broke :)

Reply:
Lets keep it for separate PR then.